tests/datalab/datalab/test_datalab.py

```diff
@@ -50,6 +50,38 @@ def test_datalab_invalid_datasetdict(dataset, label_name):
         Datalab(datadict, label_name)  # type: ignore
         assert "Please pass a single dataset, not a DatasetDict." in str(e)
 
+def check_issues_dtypes(lab):
+    """
+    Asserts that each column in lab.issues has the expected dtype.
+    
+    """
+
+    for col in lab.issues.filter(like="is_").columns:
+        assert lab.issues[col].dtype.kind == 'b', f"Column '{col}' must be boolean."
+
+    for col in lab.issues.filter(like="_score").columns:
+        assert lab.issues[col].dtype.kind in {'i', 'f'}, f"Column '{col}' must be numeric."
+
+    for col in ["given_label", "predicted_label", "distance_to_nearest_neighbor"]:
+        if col in lab.issues.columns:
+            series = lab.issue_summary[col]
+            dtype = series.dtype
+            assert np.issubdtype(dtype, np.number), (
+                f"Column '{col}' must be numeric"
+            )
+
+def check_issue_summary_dtypes(lab):
+    """
+    Asserts that each column in lab.issue_summary has the expected dtype.
+    
+    """
+    
+    for col in ["score", "num_issues"]:
+        series = lab.issue_summary[col]
+        dtype = series.dtype
+        assert np.issubdtype(dtype, np.number), (
+            f"Column '{col}' must be numeric"
+        )
```

Comment:
seems like in the code, you're always calling both 

```
check_issues_dtypes(lab)
check_issue_summary_dtypes(lab)
```
so I'd make another helper method that has both of these in it, and call this new helper throughout to simplify the code.